dpnp/dpnp_iface_sorting.py

```diff
@@ -101,6 +106,36 @@ def argsort(in_array1, axis=-1, kind=None, order=None):
     return numpy.argsort(in_array1, axis, kind, order)
 
 
+def partition(arr, kth, axis=-1, kind='introselect', order=None):
```

Comment:
use `self` for accessing the attributes and methods of the instance of the class.